astropy/constants/codata2010.py

```diff
@@ -0,0 +1,113 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Astronomical and physics constants in SI units.  See :mod:`astropy.constants`
+for a complete listing of constants defined in Astropy.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import numpy as np
+
+from .constant import Constant, EMConstant
+
+
+# PHYSICAL CONSTANTS
+
+class CODATA2010(Constant):
+    default_reference = 'CODATA 2010'
+    _registry = {}
+    _has_incompatible_units = set()
+
+    def __new__(cls, abbrev, name, value, unit, uncertainty,
+                reference=default_reference, system=None):
+        return(super(CODATA2010, cls).__new__(cls, abbrev, name, value, unit, uncertainty,
```

Comment:
(very minor nitpick) - this line is over 80 chars - maybe line break after ``unit,`` instead of ``uncertainty,``?